git_machete/utils.py

```diff
@@ -195,9 +195,9 @@ def shell_escape(arg: str) -> str:
     return " ".join(env_repr + [cmd] + list(map(shell_escape, args)))
 
 
-def warn(msg: str, apply_fmt: bool = True) -> None:
+def warn(msg: str, apply_fmt: bool = True, end: str = '\n') -> None:
     if msg not in displayed_warnings:
-        print(colored("Warn: ", AnsiEscapeCodes.RED) + (fmt(msg) if apply_fmt else msg), file=sys.stderr)
+        print(colored("\nWarn: ", AnsiEscapeCodes.ORANGE) + (fmt(msg) if apply_fmt else msg), file=sys.stderr, end=end)
```

Comment:
Hmmm is the `end` param ever provided with a non-default value in the codebase?

Reply:
@PawelLipski It will be in https://github.com/VirtusLab/git-machete/pull/604